### DISCUSSION TITLE
I cannot proceed to payment as a company beacuse of missing region

### DISCUSSION THREAD

**Discussion by walva** [community/community#67291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

The form for sponsor payment seems to be broken. I live in Belgium and I try to pay some sponsorship with my company because I use @symfony a lot. I cannot select any "region" because the result field is empty.

![image](https://github.com/community/community/assets/3897363/b7f5825d-116b-42f3-b68e-b82efc129dbd)

![image](https://github.com/community/community/assets/3897363/693121ac-9b29-48a0-b310-65d453f54f70)

I tried to edit manually the field by myself but it didn't work.

![image](https://github.com/community/community/assets/3897363/bc1cde30-41f5-4885-8943-918f7575ed3f)

I tried to remote the "required" from the field, it didn't work neither.

![image](https://github.com/community/community/assets/3897363/5a823075-0319-4f69-baa9-ebe011dc93fc)

Is this the best place to report this issue ?

**Comment by DHayes10** [comment]
Hi @walva, 

We are sorry that you're unable to add your region for payment purposes. Are you running into the tax info issue in Stripe? 

Best, 
GitHub Sponsors

**Reply by walva** [reply]
Hi @DHayes10,

No. Should I ? I don't understand what I could to, it is most likely an integration issue.

**Reply by DHayes10** [reply]
Hi @walva Hainaut is a valid region in Belgium, so you should be able to enter that in the field above. Where are you getting this error message? Is it in your GitHub profile?
